dev/ci/ci-basic-overlay.sh

```diff
@@ -29,13 +29,16 @@
 ########################################################################
 # Unicoq + Mtac2
 ########################################################################
-# Latest commit on master as of Sep 27, 2018
-: "${unicoq_CI_REF:=1cec038ab34e03109f5587a8aecda1f6c53495dd}"
+# There is no (apparent) 8.9 branch
```

Comment:
There is a tag: v1.3-8.9 (found in https://github.com/unicoq/unicoq/releases). I'd recommend using such a tag when available.

Reply:
Thanks for the hint! I will also include the link to the release page for next time.

Do you have an opinion on if a 8.9 branch should be used if there is one? I think we once discussed that releases should only use tags and hashes, but I am not so sure when the release will be and planned to replace the branches with current tags / hashes shortly before the release and change it back later.

Reply:
I wouldn't mind using the v8.9 branch when it exists instead of a commit hash / tag. But note that @silene plans to release 8.9.1 soon anyway (possibly as soon as next week).